internal/plugin/service/organization/organization_application_user.go

```diff
@@ -143,21 +143,23 @@ func (r *organizationApplicationUser) fillModel(
 		return err
 	}
 
-	var appUser *aiven.ApplicationUserInfo
+	var appUser aiven.ApplicationUserInfo
 
 	for _, u := range appUsers.Users {
 		if u.UserID == model.UserID.ValueString() {
-			appUser = &u
+			appUser = u
 			break
 		}
 	}
 
-	if appUser == nil {
-		return fmt.Errorf(errmsg.AivenResourceNotFound, r.TypeName(), model.ID.ValueString())
+	if appUser.UserID == "" {
```

Comment:
looks like this isn't needed, could we please resort back to the original way it's done via pointer?